canary_testspec.yml

```diff
@@ -0,0 +1,30 @@
+version: 0.2
```

Comment:
I was thinking that we have this reused for all canaries since it does not include some steps in testspec.yml (which are unnecessary) but I can rename accordingly.